INSTALL.md

```diff
@@ -61,18 +61,29 @@ therefore not necessary.
 Building everything from source
 ===============================
 
-PySCIPOpt requires [Cython](http://cython.org/), at least version 0.21
-(`pip install cython`). Furthermore, you need to have the Python
+Recommended is to install in a virtual environment (e.g. python3 -m venv <DIR_PATH>).
+Please note that a globally installed version of PySCIPOpt on your machine might lead to problems.
+
+After setting up and activating your virtual environment (source <DIR_PATH>/bin/activate) make sure you have [Cython](http://cython.org/) installed, at least version 0.21
+
+    pip install cython
+
+Furthermore, you need to have the Python
 development files installed on your system (error message "Python.h not
 found"):
 
     sudo apt-get install python-dev   # for Python 2, on Linux
     sudo apt-get install python3-dev  # for Python 3, on Linux
 
-After setting up `SCIPOPTDIR` as specified above, please run
+After setting up `SCIPOPTDIR` as specified above install pyscipopt
 
+    export SCIPOPTDIR=/path/to/scip/install/dir
     python -m pip install [-e] .
 
+For recompiling the source in the current directory `.` use
+
+    python -m pip install --compile .
+
```

Comment:
```
  Building wheel for PySCIPOpt (setup.py) ... error
  ERROR: Command errored out with exit status 1:
   command: /home/usr/venv/bin/python -u -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'/tmp/pip-req-build-ra1gyhch/setup.py'"'"'; __file__='"'"'/tmp/pip-req-build-ra1gyhch/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' bdist_wheel -d /tmp/pip-wheel-aqmopeaw
       cwd: /tmp/pip-req-build-ra1gyhch/
  Complete output (8 lines):
  Using include path <path/to/scip/install/dir/include>.
  Using SCIP library <scip> at </path/to/scip/install/dir/lib>.
  usage: setup.py [global_opts] cmd1 [cmd1_opts] [cmd2 [cmd2_opts] ...]
     or: setup.py --help [cmd1 cmd2 ...]
     or: setup.py --help-commands
     or: setup.py cmd --help
  
  error: invalid command 'bdist_wheel'
  ----------------------------------------
  ERROR: Failed building wheel for PySCIPOpt
  Running setup.py clean for PySCIPOpt
Failed to build PySCIPOpt
Installing collected packages: PySCIPOpt
  Attempting uninstall: PySCIPOpt
    Found existing installation: PySCIPOpt 3.0.4
    Not uninstalling pyscipopt at /home/usr/git/PySCIPOpt, outside environment /home/usr/venv
    Can't uninstall 'PySCIPOpt'. No files were found to uninstall.
    Running setup.py install for PySCIPOpt ... done
Successfully installed PySCIPOpt-3.0.4
[edited paths]
```
This is inside a new virtualenvironment.
edit: though this error is given, the py.test is successful for feature_test as well

Reply:
you are missing `wheel`, try running `pip install wheel` before